.github/workflows/docker-publish.yml

```diff
@@ -7,14 +7,14 @@ name: Docker
 
 on:
   push:
-    branches: [ "production" ]
+    branches: [ "main" ]
```

Comment:
Another. simple update in github setting. Change default branch from main to stage, so that all future features and PR targets stage by default.